openmetadata-ui/src/main/resources/ui/src/pages/CustomizablePage/CustomizablePage.tsx

```diff
@@ -53,27 +63,40 @@ export const CustomizablePage = () => {
     setSaveCurrentPageLayout(value);
   }, []);
 
-  const fetchDocument = async () => {
-    const pageLayoutFQN = `${EntityType.PERSONA}.${personaFQN}.${EntityType.PAGE}.${pageFqn}`;
+  const fetchPersonaDetails = useCallback(async () => {
     try {
-      setIsLoading(true);
-      const pageData = await getDocumentByFQN(pageLayoutFQN);
-
-      setPage(pageData);
-      setEditedPage(pageData);
-    } catch (error) {
-      if ((error as AxiosError).response?.status === ClientErrors.NOT_FOUND) {
-        setPage({
-          name: `${decodedPersonaFQN}${decodedPageFQN}`,
-          fullyQualifiedName: getDecodedFqn(pageLayoutFQN),
-          entityType: EntityType.PAGE,
-          data: {
-            page: { layout: customizePageClassBase.defaultLayout },
-          },
-        });
-      }
+      setIsPersonaLoading(true);
+      const response = await getPersonaByName(personaFQN);
+
+      setPersonaDetails(response);
     } finally {
```

Comment:
Catch block is missing here.

Reply:
@Sachin-chaurasiya There is no operation to perform in the catch block, hence I haven't added that to avoid unnecessary lines

Reply:
@aniketkatkar97, we should add the catch block with just some comments.